src/molecules/header-card/header-card.tsx

```diff
@@ -34,16 +35,40 @@ const HeaderCardContainer = styled.div`
   border-bottom: ${(props) => `1px solid ${props.theme.color.black}`};
 `;
 
+const StyledWrapper = styled.figcaption`
+  display: flex;
+  align-items: center;
+  max-height: 90px;
```

Comment:
why are we hardcoding this? what if the caption is too long?

Reply:
will remove this